Assets/Scripts/Items/building/Building UI/BuildMenu.cs

```diff
@@ -1,12 +1,24 @@
 using Examen.Inventory;
+using Examen.Player.PlayerDataManagement;
 using UnityEngine;
 
 namespace Examen.Building.BuildingUI
 {
     public class BuildMenu : MonoBehaviour
     {
+        [SerializeField] private BuildItem[] _buildItems;
         [SerializeField] private GameObject _inventory;
 
+        private void Awake()
+        {
+            foreach (BuildItem buildItem in _buildItems)
+                buildItem.gameObject.SetActive(false);
```

Comment:
```suggestion
            foreach (BuildItem buildItem in _buildItems)
                buildItem.gameObject.SetActive(false);
```
Can you call UpdateBuildMenu from here? it is weird to start with out buildings even though you have levels

Reply:
No because the exp goes trough the server so if there is a delay to the server you can already build not unlocked buildings